scripts/install.sh

```diff
@@ -0,0 +1,445 @@
+      log ""
+      log "Failed to complete TLS handshake. Please ensure your system's TLS library is up-to-date (OpenSSL, GnuTLS, libressl, etc.)"
+    fi
+    clean_exit 1
+  fi
+
+
+  # this could be >255, so print HTTP status code rather than using as return code
+  echo "$status_code"
+}
+
+# note: wget does not retry on 5xx
+wget_download() {
+  url="$1"
+  output_file="$2"
+  component="$3"
+
+  security_flags="--secure-protocol=TLSv1_2 --https-only"
+  # determine if using BusyBox wget (bad) or GNU wget (good)
+  (wget --help 2>&1 | head -1 | grep -iv busybox > /dev/null 2>&1) || security_flags=""
+  # only print this warning once per script invocation
+  if [ -z "$security_flags" ] && [ "$component" = "Binary" ]; then
+    log_debug "Skipping additional security flags that are unsupported by BusyBox wget"
+    # log to stderr b/c this function's stdout is parsed
+    log_warning "This system's wget binary is provided by BusyBox. Doppler strongly suggests installing GNU wget, which provides additional security features."
+  fi
+
+  # allow wget to fail w/o exiting
+  set +e
+  # we explicitly disable shellcheck here b/c security_flags isn't parsed properly when quoted
+  # shellcheck disable=SC2086
+  headers=$(wget $security_flags -q -t 5 -S -O "$output_file" "$url" 2>&1)
+  exit_code=$?
+  set -e
+
+  status_code="$(echo "$headers" | grep -o -E '^\s*HTTP/[0-9.]+ [0-9]{3}' | tail -1 | grep -o -E '[0-9]{3}')"
+  # it's possible for this value to be blank, so confirm that it's a valid status code
+  valid_status_code=0
+  if expr "$status_code" : '[0-9][0-9][0-9]$'>/dev/null; then
+    valid_status_code=1
+  fi
+
+  if [ "$exit_code" -ne 0 ]; then
+    if [ "$valid_status_code" -eq 1 ]; then
+      # print the code and continue
+      log_debug "Request failed with http status $status_code"
+      log_debug "Response headers:"
+      log_debug "$headers"
+    else
+      # exit immediately
+      log "ERROR: wget failed with exit code $exit_code"
+
+      if [ "$exit_code" -eq 5 ]; then
+        log ""
+        log "Ensure the ca-certificates package is installed for your distribution"
+      fi
+      clean_exit 1
+    fi
+  fi
+
+
+  # this could be >255, so print HTTP status code rather than using as return code
+  echo "$status_code"
+}
+
+check_http_status() {
+  status_code="$1"
+  component="$2"
+
+  if [ "$status_code" -ne 200 ]; then
+    error="ERROR: $component download failed with status code $status_code."
+    if [ "$status_code" -ne 404 ]; then
+      error="${error} Please try again."
+    fi
+
+    log ""
+    log "$error"
+
+    if [ "$status_code" -eq 404 ]; then
+      log ""
+      log "Please report this issue:"
+      log "https://github.com/DopplerHQ/cli/issues/new?template=bug_report.md&title=[BUG]%20Unexpected%20404%20using%20CLI%20install%20script"
```

Comment:
Why is doppler here?

Reply:
Because it was adapted from their script. Removed in https://github.com/seamapi/seam-cli/pull/65